aiogram/filters/command.py

```diff
@@ -89,6 +100,9 @@ def __init__(
                 )
             if ignore_case and isinstance(command, str):
                 command = command.casefold()
+            for individual_prefix in prefixes:
+                if isinstance(command, (str, BotCommand, re.Pattern)):
```

Comment:
because of previous checks, command here can only be `str`, `BotCommand` or `re.Pattern`, so, you don't need to check it additionally.

BotCommand is also impossible case, because o first condition in this loop.